### PR TITLE
Documentation for list/add/edit patient medical records

### LINKED ISSUE #19: Diego documentation

### PR DESCRIPTION
**Pull request by jmsandiegoo**
Resolves #19 

Added documentation for:
- #8 
- #9 
- #10

### CODE REVIEW

**Review by raysonchia — APPROVED** [review]
LGTM!

File: docs/AboutUs.md [hunk 1]
```diff
@@ -55,5 +55,5 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 [[github](https://github.com/joenus)]
 [[portfolio](team/joe.md)]
 
-* Role: Developer
+* Role: Team Lead, Developer
```
Comment (joenus) [inline docs/AboutUs.md:58]: 💀💀💀

File: docs/images/jmsandiegoo.png
Comment (joenus) [inline docs/images/jmsandiegoo.png:1]: Handsome boi

**Review by joenus — APPROVED** [review]

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -262,23 +262,28 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| Priority | As a …​                                    | I want to …​                          | So that I can…​                                          |
+| -------- | ------------------------------------------ |---------------------------------------|----------------------------------------------------------|
+| `* * *`  | user                                       | view all my patient's medical records | have a clear overview of all my records                  |
+| `* * *`  | user                                       | add a patient's medical record     | record new patients I work with                          |
+| `* * *`  | user                                       | edit a patient's medical record     | ammend necessary details of a medical record when needed |
+| `* * *`  | user                                       | add a patient's appointment note      | record keep the details of each appointment              |
+| `* * *`  | user                                       | delete a patient's appointment note   | remove entries that I no longer need                     |
```
Comment (sebin0817) [inline docs/DeveloperGuide.md:287]: Should add MSS details for your patient record use cases

**Pull request merged**